packages/input/src/index.ts

```diff
@@ -78,6 +80,14 @@ export default createPrompt<string, InputConfig>((config, done) => {
     }
   });
 
+  // If prefill is set to 'editable' cut out the default value and paste into current state and the user's cli buffer
+  // They can edit the value immediately instead of needing to press 'tab'
+  useEffect((rl) => {
+    if (prefill !== 'editable' || !defaultValue) return;
```

Comment:
nit, let's wrap the logic inside the conditional instead of the early return (because it's an effect handler, it'll keep it extendable for other behaviors.)